README.md

```diff
@@ -16,6 +16,15 @@ npm install --save-dev @datadog/datadog-ci
 yarn add --dev @datadog/datadog-ci
 ```
 
+If you need `datadog-ci` as a CLI tool instead of a package, you can install it globally:
+
+```sh
+# NPM
+npm install -g @datadog/datadog-ci
+
+# Yarn
+yarn global add @datadog/datadog-ci
```

Comment:
actually i'm not really sure if these are the best practices, maybe i should mention using `npx`?